lib/jormungandr/test/data/jormungandr/genesis.yaml

```diff
@@ -3062,3 +3062,14 @@ initial:
   - fund:
     - address: ca1swl53wlqt5dnl63e0gnf8vpazgt6g5mq384dmz72329eh4m8z7e5un8q6lg
       value: 100000000000
+
+  # Funds & certificates for the single stake pool
```

Comment:
For the sake of testing listing of stake pools maybe it would be worth adding a few stake pools, with different initial stakes. Say 3 stake pools (high stake, moderate stake and very low stake).

Reply:
Good idea. But, yes, I think we can delay it to the ticket where it will first be useful.